js/samples/04.ai-apps/d.assistants-mathBot/infra/azure.parameters.json

```diff
@@ -17,8 +17,11 @@
         "botDisplayName": {
             "value": "MathBot"
         },
-        "openAIKey": {
```

Comment:
same with keeping this but commenting it out?

Reply:
Can't comment out in json files